### DISCUSSION TITLE
Reminder Default time

### DISCUSSION THREAD

**Discussion by steve-alpoz** [freeplane/freeplane#1877] [General]
When adding a reminder (CTRL+T) I think it would be beneficial for a lot of users to have an option to set a default reminder Time. 
At the moment the time that is presented in the dialogue is the current system time.  And I have to manually correct it every time.
I always set my reminders to 9am. 
If there was a field in settings, if this field had a value then that value could be used, and if it was empty then the value could default to the current time for that dialogue like it presently does.
.... or is there already a setting and I have just missed it?

**Comment by macmarrum** [comment]
The time in `Tools->Manage time…` is hard-coded to current time.

Here's a script that should cover your needs.

1. Save [createOrReplaceReminder.groovy](https://raw.githubusercontent.com/macmarrum/freeplane-scripting/main/src/scripts/createOrReplaceReminder.groovy) to `<user-dir>/scripts`
2. You can adjust the menu location of the script by changing or removing `="/menu_bar/Mac1/Create"`
3. Restart Freeplane
4. Optionally, remap `Ctrl+T` to launch the script, instead of `Tools->Manage time…`

_**Note**: You can find your `<user-dir>` in Freeplane `Tools -> Open user directory`_

**Comment by steve-alpoz** [comment]
Thanks @macmarrum. You are a wizard. It works perfectly, and your instructions were easy to follow.
